### ISSUE TITLE
i18n / l10n

### ISSUE THREAD

**Issue by bforest** [teqnology/alfresco-login-reset-share#2]
Hi,

It will be interesting to use i18n / l10n mechanisms (.properties files).
Your modules cannot be translated properly by no english speakers.

Regards.

**Comment by teqnology** [comment]
I agree, it's a work in progress, for now the only thing I could recommend you, you can easily navigate to the templates folder in your share instance:
https://github.com/teqnology/alfresco-login-reset-share/tree/master/src/main/amp/config/alfresco/templates/com/androgogic/base/login
and update the HTML code to reflect your language. Also, the email templates should be handled as well. This will be my next step while improving the extension. Thanks for the heads up :)
